lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -6460,7 +6460,7 @@ Expected<std::unordered_map<Node *, ConcatNode *>> glow::parallelizeOps(
       case Kinded::Kind::BatchedReduceAddNodeKind: {
         BatchedReduceAddNode *BR = llvm::cast<BatchedReduceAddNode>(curNode);
         splitDims[BatchedReduceAddNode::BatchIdx] =
-            (BR->getAxis() == 0) ? 1 : 0;
+            (BR->getAxes()[0] == 0) ? 1 : 0;
```

Comment:
Can you also add a check above here that axes size is 1 too, using `RETURN_ERR_IF_NOT()`, e.g. `RETURN_ERR_IF_NOT(BR->getAxes().size() == 1, "Parallelization for BatchedReduceAdd only supported for single axis);`